spack.yaml

```
@@ -6,57 +6,64 @@
 # https://github.com/ACCESS-NRI/model-deployment-template/blob/main/spack.yaml
 spack:
   specs:
-    - access-om3@git.2025.08.001
+    - access-om3@git.2025.08.002
```

```suggestion
    - access-om3@git.2025.08.002 /sayts7gqb
```
Re-use the ESMF build created in this PR.